starter-code/js/index.js

```diff
@@ -11,8 +24,14 @@ function updatePriceByProduct(productPrice, index){
 }
 
 function getTotalPrice() {
+  var totalItems = Number(document.querySelectorAll(".priceforone").innerText);
```

Comment:
document.querySelectorAll(".priceforone") will return an array. You can not do a innerText of an array.